content/guides/getstarted/hello-world-in-your-browser.md

```diff
@@ -176,22 +173,24 @@ behalf to ensure your wallet has enough SOL to cover the cost of deployment.
 solana airdrop 2
 ```
 
-<!-- ![Build and deploy your Solana program to the blockchain](/img/quickstarts/solana-get-started-build-and-deploy.png) -->
+> Note: If you get a rate limit error when requesting devnet SOL you can also
+> you can just type `run` in the terminal and you will get a link to a
+> [web faucet](https://faucet.solana.com/). You can also find your wallet
+> address at the bottom of the playground window.
```

Comment:
markdown blockquotes are auto-converted to the blue "info" box. So writing "note" is now redundant

```suggestion
> If you get a rate limit error when requesting devnet SOL you can also you can
> just type `run` in the terminal and you will get a link to a
> [web faucet](https://faucet.solana.com/). You can also find your wallet
> address at the bottom of the playground window.
```